src/Illuminate/Support/Testing/Fakes/EventFake.php

```diff
@@ -211,7 +211,7 @@ public function dispatch($event, $payload = [], $halt = false)
         if ($this->shouldFakeEvent($name, $payload)) {
             $this->events[$name][] = func_get_args();
         } else {
-            $this->dispatcher->dispatch($event, $payload, $halt);
+            return $this->dispatcher->dispatch($event, $payload, $halt);
```

Comment:
Why not an early return?
```php
if ($this->shouldFakeEvent($name, $payload) === false) {
    return $this->dispatcher->dispatch($event, $payload, $halt);
}

$this->events[$name][] = func_get_args();
```

Reply:
I was trying to make as small a change as possible.